packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -249,15 +242,15 @@ class _TreeNodeState<T extends TreeNode<T>> extends State<_TreeNodeWidget<T>>
       curve: Curves.easeInOutCubic,
       parent: showController,
     );
-    expandAnimation = Tween<double>(begin: 0.0, end: 0.5).animate(
+    expandAnimation = Tween<double>(begin: 0.75, end: 1.0).animate(
```

Comment:
comment why these magic constants are 0.75 and 1.0

Reply:
Done.